cJSON.c

```diff
@@ -2726,7 +2726,14 @@ CJSON_PUBLIC(cJSON *) cJSON_CreateStringArray(const char *const *strings, int co
 }
 
 /* Duplication */
+cJSON * cJSON_Duplicate_rec(const cJSON *item, size_t depth, cJSON_bool recurse);
+
 CJSON_PUBLIC(cJSON *) cJSON_Duplicate(const cJSON *item, cJSON_bool recurse)
+{
+    return cJSON_Duplicate_rec(item, 0, recurse );
+}
+
+cJSON * cJSON_Duplicate_rec(const cJSON *item, size_t depth, cJSON_bool recurse)
```

Comment:
Looks the input `cJSON_bool recurse` is never used.

Reply:
Indeed. It was added in the initial commit of cJSON_Duplicate (927aa631b82143ab5304087446f173819eb94543). It's also part of the of public API. Happy to remove it but I thought I would stick to a simple fix.

Reply:
Ah, `recurse` was used [here](https://github.com/DaveGamble/cJSON/blob/master/cJSON.c#L2779).
I missed it. My bad.